server/boot/routes.js

```diff
@@ -29,12 +30,23 @@ module.exports = function(app) {
       password: req.body.password
     }, 'user', function(err, token) {
       if (err) {
-        res.render('response', {
-          title: 'Login failed',
-          content: err,
-          redirectTo: '/',
-          redirectToLinkText: 'Try again'
-        });
+        if(err.details != null){
```

Comment:
`err.details` is a generic field that may be provided by many different errors. To handle the case when the user was not verified, you should check:

```js
if (err.details && err.details.code === 'LOGIN_FAILED_EMAIL_NOT_VERIFIED')
```

To keep the size of this patch (diff) smaller, I'd recommend to use an early return in the new if block.


```js
if (err.details && err.details.code === 'LOGIN_FAILED_EMAIL_NOT_VERIFIED')
  res.render('reponseToTriggerEmail', {
    // ...
  });
  return;
}

res.render('response', {
  // ...
});
return;
```